Libraries/LibGUI/TextEditor.h

```diff
@@ -248,6 +254,9 @@ class TextEditor
     bool m_automatic_indentation_enabled { false };
     bool m_line_wrapping_enabled { false };
     bool m_readonly { false };
+    bool m_displayonly { false };
```

Comment:
Since these two can be combined in invalid ways, it would be nicer to use an enum for the mode. :)